ocaml/xapi/xapi_network_sriov_helpers.ml

```diff
@@ -0,0 +1,171 @@
+(*
+ * Copyright (C) 2017 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Stdext
+open Listext
```

Comment:
Please do not do that. Instead do:
```
open Xapi_stdext_std
```

and then in the code prepend `Listext` only when needed.

To be honest I don't see any use of `Stdext` in this module, so you should actually remove these two lines.